src/ZkFold/Symbolic/Algorithms/Hash/MiMC/Constants.hs

```diff
@@ -1,21 +1,235 @@
 module ZkFold.Symbolic.Algorithms.Hash.MiMC.Constants (mimcConstants) where
 
-import           Crypto.Hash.SHA256              (hash)
-import           Data.Maybe                      (fromJust)
 import           Prelude
 
 import           ZkFold.Base.Algebra.Basic.Class (FromConstant (..))
-import           ZkFold.Base.Data.ByteString
 
-mimcSeed :: LittleEndian
-mimcSeed = 42
+cs :: [Integer]
+cs = [42,
```

Comment:
Why are we making this change?

Reply:
For speed. The list of constants was generated every time mimcHash was called. It took too long. This list is just the hardcoded result of the constants generator

Reply:
Can it be a `take 218 $ map getI $ iterate hash $ toByteString mimcSeed` at least? Maybe GHC would compute it only once this way